mingw-cross/Makefile

```diff
@@ -1,25 +1,22 @@
-# Makefile for Quetoo Windows distributable.
```

Comment:
@Paril If there's something you're looking for as a guide, I guess the MinGW Makefile would be the most appropriate thing.